src/Middleware/StaticFiles/samples/StaticFileSample/StaticFileSample.csproj

```
@@ -0,0 +1,14 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
```

```suggestion
    <TargetFramework>netcoreapp2.1</TargetFramework>
```

The original might be netcoreapp2.0, but let's take this chance to cleanup.